setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='mongodb-migrations',
-    version='0.4.1',
+    version='0.5.0',
```

Comment:
I've bumped the version to `0.5.0` as requested.

Reply:
nice!